test/node/reducers/vis-state-test.js

```diff
@@ -1186,6 +1186,175 @@ test('#visStateReducer -> DUPLICATE_LAYER', t => {
   t.end();
 });
 
+// TODO: not sure if this test is correct, it basically checks if the loaded visState is empty
+// if we load configuration through the schema manager all layers will be changing from having visualChannels to visualConfig
```

Comment:
Not sure where the test comes from, but maybe the goal is to make sure that an invalid layer wouldn't be loaded?